components/Header.js

```diff
@@ -0,0 +1,24 @@
+import IPlantLogo from "@/public/iplant-logo.svg";
+import styled from "styled-components";
+import Link from "next/link";
+
+export default function Header() {
+  return (
+    <StyledHeader>
+    <Link href={"/"}> 
```

Comment:
please include an 'aria-label' for each button or link that has no text content.